packages/layer-composer/src/generators/heatmap/config.ts

```diff
@@ -9,17 +9,16 @@ export const API_ENDPOINTS = {
 
 export const HEATMAP_DEFAULT_MAX_ZOOM = 12
 
-export const HEATMAP_MODE_COMBINATION: Record<
-  HeatmapAnimatedMode,
-  HeatmapAnimatedCombinationMode
-> = {
-  [HeatmapAnimatedMode.Compare]: HeatmapAnimatedCombinationMode.Compare,
-  [HeatmapAnimatedMode.Bivariate]: HeatmapAnimatedCombinationMode.Bivariate,
-  [HeatmapAnimatedMode.Blob]: HeatmapAnimatedCombinationMode.Literal,
-  [HeatmapAnimatedMode.Extruded]: HeatmapAnimatedCombinationMode.Cumulative,
+export const HEATMAP_MODE_COMBINATION: Record<HeatmapAnimatedMode, SublayerCombinationMode> = {
+  [HeatmapAnimatedMode.Single]: SublayerCombinationMode.none,
+  [HeatmapAnimatedMode.Compare]: SublayerCombinationMode.max,
+  [HeatmapAnimatedMode.Bivariate]: SublayerCombinationMode.bivariate,
+  [HeatmapAnimatedMode.Blob]: SublayerCombinationMode.literal,
+  [HeatmapAnimatedMode.Extruded]: SublayerCombinationMode.cumulative,
```

Comment:
Not sure if I missed something yesterday from conversation about enums.
Are we using Enum.Property in uppercase or lowercase ? just to keep the same pattern for next things

Reply:
Whatever you prefer, both looks to me.
I'd say uppercase as there are lot of places already using it